_data/projects/ElectronClippy.yml

```diff
@@ -0,0 +1,10 @@
+name: Clippy
+desc: Electron App that records your Clipboard history and Image history
+site: https://github.com/Jarmahent/Clippy
+tags:
+- electron
+- javascript
+- react
+upforgrabs:
+  name: Help Wanted
```

Comment:
@Jarmahent Just one last thing you have missed. "Help Wanted" and "help wanted" isn't the same thing so please change this to "help wanted" as your label is "help wanted" not "Help Wanted"